static/src/js/components/SavedProjects/__tests__/SavedProjects.test.jsx

```diff
@@ -1,142 +1,130 @@
-          created_at: '2019-08-25T11:58:14.390Z'
-        }],
-        savedProjectsIsLoaded: true
+    test('renders the correct collection count and time-created cell', () => {
+      setup({
+        overrideProps: {
+          isLoaded: true,
+          projects: [project]
+        }
       })
 
-      expect(enzymeWrapper.find(Table).length).toBe(1)
-      expect(enzymeWrapper.find('tbody tr').length).toBe(1)
-      expect(enzymeWrapper.find(PortalLinkContainer).prop('to')).toEqual('/search?projectId=8069076')
-      expect(enzymeWrapper.find(PortalLinkContainer).prop('children')).toEqual('test name')
-      expect(enzymeWrapper.find('tbody tr td').at(1).text()).toEqual('2 Collections')
+      expect(screen.getByText('1 Collection')).toBeInTheDocument()
+      expect(screen.getByText(/ago$/)).toBeInTheDocument()
     })
+  })
 
-    test('renders a table when a saved project exists with one collection without a name', () => {
-      const { enzymeWrapper } = setup({
-        savedProjects: [{
-          id: '8069076',
-          name: '',
-          path: '/search?p=!C123456-EDSC',
-          created_at: '2019-08-25T11:58:14.390Z'
-        }],
-        savedProjectsIsLoaded: true
+  describe('When deleting a project', () => {
+    const project = {
+      id: '8069076',
+      name: 'test project',
+      path: '/search?p=!C123456-EDSC',
+      created_at: '2019-08-25T11:58:14.390Z'
+    }
+
+    test('calls onDeleteProject when delete button is clicked', async () => {
+      const mockOnDeleteProject = jest.fn()
+
+      const { user } = setup({
+        overrideProps: {
+          isLoaded: true,
+          projects: [project],
+          onDeleteProject: mockOnDeleteProject
+        }
       })
 
-      expect(enzymeWrapper.find(Table).length).toBe(1)
-      expect(enzymeWrapper.find('tbody tr').length).toBe(1)
-      expect(enzymeWrapper.find(PortalLinkContainer).prop('to')).toEqual('/search?projectId=8069076')
-      expect(enzymeWrapper.find(PortalLinkContainer).prop('children')).toEqual('Untitled Project')
-    })
-
-    test('deleting a project calls onDeleteSavedProject', () => {
-      const { enzymeWrapper, props } = setup({
-        savedProjects: [{
-          id: '8069076',
-          name: 'test name',
-          path: '/search?p=!C123456-EDSC',
-          created_at: '2019-08-25T11:58:14.390Z'
-        }],
-        savedProjectsIsLoaded: true
+      const deleteButton = screen.getByRole('button', {
+        name: /remove project/i
       })
-      window.confirm = jest.fn(() => true)
-
-      const button = enzymeWrapper.find('.saved-projects__button--remove')
-      button.simulate('click')
 
-      expect(window.confirm).toBeCalledTimes(1)
-      expect(props.onDeleteSavedProject).toBeCalledTimes(1)
-      expect(props.onDeleteSavedProject).toBeCalledWith('8069076')
+      await user.click(deleteButton)
+      expect(mockOnDeleteProject).toHaveBeenCalledWith('8069076')
```

Comment:
Any time you have the `toHaveBeenCalledWith` assertion you also need to include `toHaveBeenCalledTimes` (and vice versa). Checking that it is called the right number of times is important to catch if things are being called more times than we expect.